packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -46,7 +46,7 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
 
     const { isFocused } = this.state;
     const isChecked = checked === undefined ? this.state.isChecked : checked;
-
+    const inputName = (inputProps && inputProps.name) ? inputProps.name : this._id;
```

Comment:
I don't think this is what I expected. i expected:

```ts
<Checkbox name='foo' />
```

The `inputProps` is a really rather unfortunate "backdoor" prop that should be avoided if there is something more expected or ergonomic solution.